client/src/pages/AddProject.js

```diff
@@ -0,0 +1,279 @@
+import React, { Component } from 'react';
+import { Typography, withStyles, Button, TextField } from '@material-ui/core';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import validator from 'validator';
+
+import { OutlinedSelect } from '../components/Inputs';
+import UploadImages from '../components/UploadImages';
+import FormValidator from '../helpers/form-validation';
+import { withToast } from '../components/Toast';
+import { addProject } from '../api/projects';
+
+const styles = {
+    pageContent: {
+        display: 'flex',
+        justifyContent: 'left'
+    },
+    projectPreviewContainer: {
+        display: 'flex',
+        width: '25%',
+        flexDirection: 'column',
+        margin: '0px 20px 0px 10px',
+        padding: '30px 50px 20px 30px',
+        boxShadow: '2px 0px 4px 2px #D3D3D3',
+    },
+    addProjectPage: {
+        margin: '10px',
+        padding: '20px',
+    },
+    form: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'left',
+        width: '100%',
+        marginTop: '30px'
+    },
+    formLine: {
+        marginBottom: '20px',
+    },
+    button: {
+        margin: '20px 0px 10px 0px'
+    }
+}
+
+const industries = [
```

Comment:
Maybe the least pressing issue of all, but maybe these can live somewhere as constants or something like that?

Reply:
Yeah. It's really just dummy data since I don't really know how we will handle lists of locations and industries in the future. Industries seem much more manageable but locations can get pretty huge. I'll take your suggestion and move them to a separate folder and file

Reply:
okay -- I thought they were constants. Good point about handling location. That will definitely be an interesting challenge.

Reply:
Yeah I'm curious to see how it's handled. I imagine it's just some object with locations or maybe some crazy data structure I haven't heard of yet lol